### ISSUE TITLE
activeresource with Rails 3?

### ISSUE THREAD

**Issue by dannymcc** [wildbit/beanstalkapp#1]
I have come to use the gem for the first time and keep getting activeresource errors. Is there a way of using this gem with activeresource 3.0.3, alongside Rails 3?

Bundler could not find compatible versions for gem "activeresource":
  In Gemfile:
    rails (= 3.0.3) depends on
      activeresource (= 3.0.3)

```
activeresource (2.3.5)
```

**Comment by isabanin** [comment]
Hey Danny,

For now ActiveSupport 2.3.5 is required. We will need to review the code and do some testing before we can migrate to ActiveSupport 3. I think first step towards that migration would be to write a proper test suit for the gem.

Ilya Sabanin
Lead Beanstalk Developer
http://twitter.com/isabanin

**Comment by dannymcc** [comment]
Hey Ilya,

I have forked your repo and amended the following line - will it completely fail if I do this?

// beanstalkapp.gemspec
s.add_runtime_dependency(%q<activeresource>, ["= 3.0.4"])

**Comment by isabanin** [comment]
Danny, that won't work for Rails 2 applications. And I can't guarantee that the gem will work fine with AS3.

Ilya Sabanin
Lead Beanstalk Developer
http://twitter.com/isabanin

**Comment by jeroenbourgois** [comment]
any updates?

**Comment by isabanin** [comment]
Sorry, no updates yet. We are working on a huge update for Beanstalk and don't have time for the gem currently.

Ilya Sabanin
http://twitter.com/isabanin

**Comment by MattKiazyk** [comment]
Would be awesome if this could be updated. Any updates?

**Comment by isabanin** [comment]
Sorry, no updates yet. We don't have enough time for the gem at the moment. 

---

Ilya Sabanin
http://ilya.sabanin.ru

**Comment by paulthorsteinson** [comment]
This would be nice to have fixed for sure.  We were using this but since some of our devs got this error we just ditched the gem in favour of rest-client.

**Comment by rutgerg** [comment]
Is this every going to happen?

**Comment by isabanin** [comment]
We don't have any plans to make a new version of the gem right now.
